### ISSUE TITLE
Select an ORM to integrate with

### ISSUE THREAD

**Issue by ahalls** [GaltSoft/kitura-manager#4]
We should start with one ORM

**Comment by the-freshlord** [comment]
Should do this one first before doing issue #3

**Comment by the-freshlord** [comment]
The only major ORM that I have found is StORM, but their library isn't fully complete. There is also, Swift-Kuery, which a good amount of people are using already. What do you think about building our own ORM thats built on-top of Swift-Kuery?

**Comment by ahalls** [comment]
We are not going to build an ORM as part of this project.  It will make it too huge.  If you can get support for building an ORM in the Kitura community or elsewhere on top of Swift-Kuery go for it.

If you think as a first step we should build directly on Swift-Kuery ... I'm open for further discussion.

**Comment by ahalls** [comment]
We are not going to build an ORM as part of this project.  It will make it way too huge ... 

If you can find support elsewhere to build an orm on-top of Swift-Kuery go for it.

If no adequate ORM Exists we can build on top of Swift-Kuery ...

In principle we might be saying the same thing.  But I don't want to take on the mantle "we are building an ORM". That brings on a lot of baggage.  But if we just build a thin layer of ORMishness to generate CRUD actions on top of Swift-Kuery ... that is a good approach. Later we can integrate with a full feature ORM or spin off an ORM project.

**Comment by ahalls** [comment]
https://github.com/IBM-Swift/Kitura/issues/791 

I would suggest that we discuss this on the kitura slack channel

**Comment by ahalls** [comment]
@eman6576  Have you looked at Vapor/Fluent ?

**Comment by the-freshlord** [comment]
Yes I have @ahalls. But i'm not sure if its a generic solution thats plug and play or is built around Vapor. Also, it isn't extremely documented.

**Comment by ahalls** [comment]
Decided to go forward in Swift-Kuery.
